boards/arm/mps2_an521/CMakeLists.txt

```diff
@@ -18,7 +18,8 @@ if (CONFIG_BUILD_WITH_TFM)
 		set(TFM_IMAGE_VERSION_NS 0.0.0+0)
 	endif()
 
-	set(PREPROCESSED_FILE "${CMAKE_BINARY_DIR}/tfm/image_macros_preprocessed")
+	set(PREPROCESSED_FILE_S "${CMAKE_BINARY_DIR}/tfm/bl2/ext/mcuboot/CMakeFiles/signing_layout_s.dir/signing_layout_s.o")
+	set(PREPROCESSED_FILE_NS "${CMAKE_BINARY_DIR}/tfm/bl2/ext/mcuboot/CMakeFiles/signing_layout_ns.dir/signing_layout_ns.o")
```

Comment:
I believe those two files should be be returned by the TF-M build, or exists as TF-M build properties that can be fetched when needed.

This is a bit to deep knowledge of the TF-M build structure.

Reply:
can of course be improved later, just wanted to mention it.